sandbox/install-client-sources.sh

```diff
@@ -9,6 +9,9 @@
 
 set -e
 
+# Upgrade pip to the latest version
+pip install --upgrade pip
+
```

Comment:
I don't think updating pip here is a good idea. It is not required for this script to work and it risks messing up the user environment.